content/release-notes/1.51.0.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2021-09-03T17:00:00-00:00
+linktitle: "1.51.0"
+title: "1.51.0"
+kubernetes: "1.19, 1.20, and 1.21"
+---
+
+{{<features>}}
+{{</features>}}
+
+{{<changes>}}
+{{</changes>}}
+
+{{<fixes>}}
+* Fixed a bug that caused KOTS to fail to detect private images if their domains are blocked by a firewall in online installs.
+* Fixed a bug that caused the [set config](/kots-cli/set/config/) command to fail if appliction config had a required password value that was already set.
+* Fixed a bug that caused [LocalImageName](/reference/template-functions/config-context/#localimagename) function to not require private images when `proxyPublicImages` is set.
+* Fixed a bug that could cause the secret used to pull private images to be removed during snapshot restore when multiple applications are installed.
+* Fixed a bug that could cause [Helm render](/vendor/helm/helm-processing/) chart using API not supported by the target kubernetes cluster.
+* Fixed a bug that caused newer style charts with private images failed to kustomize.
+* Fixed a bug that caused coexistence of legacy and newer style charts with private images failed to deploy.
```

Comment:
This one was super confusing to read. I re-worded above with how I understood it. Please clarify if i misunderstood :)